repo/blob/s3/s3_storage_test.go

```diff
@@ -119,7 +119,7 @@ func getProviderOptions(tb testing.TB, envName string) *Options {
 	}
 
 	if o.Prefix != "" {
-		tb.Fatalf("options providd in '%v' must not specify a prefix", envName)
+		tb.Fatalf("options provided in '%v' must not specify a prefix", envName)
```

Comment:
Note: `tb.Fatalf` doesn't actually make the test fail. For that it should be followed by a `tb.FailNow()`